cmd/clone.go

```diff
@@ -18,6 +19,12 @@ var cloneCmd = &cobra.Command{
 	Long: `Clone supports these shorthands
 - repo
 - namespace/repo`,
+	Args: func(cmd *cobra.Command, args []string) error {
```

Comment:
Can you implement something like [this](https://github.com/zaquestion/lab/blob/master/cmd/version.go#L15) here? This will allow us to pass the git clone output through. Ideally we'd want to ditch any lab/cobra output as well and make it look like we aren't involved.

Reply:
done!